apis/supernova/src/handler/data-property-handler.js

```diff
@@ -69,6 +173,105 @@ class DataPropertyHandler {
   static getAlternativeMeasures() {
     throw new Error('Method not implemented.');
   }
+
+  static addMeasure() {
+    throw notSupportedError;
+  }
+
+  static addMeasures() {
+    throw notSupportedError;
+  }
+
+  static autoSortMeasure() {
+    throw notSupportedError;
+  }
+
+  createExpressionMeasure(expression, label, defaults) {
+    const measure = merge({}, this.measureProperties || {}, defaults || {});
+
+    const meas = { ...measure };
```

Comment:
`merge()` returns a new object. Should be no need to create clone of that.

Reply:
Ah, true! I'll remove it.